lib/indexer.rb

```diff
@@ -0,0 +1,151 @@
+module Argo
+  class SimpleLogJob
+    def initialize(log_val)
+      @log_val = log_val
+    end
+
+    def perform
+      sleep(rand 10)
+
+      log_filename = "tmp/simple_job.log"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.